QUESTIONS.rst

```diff
@@ -5,20 +5,19 @@ To use the package template run ``cookiecutter gh:astropy/package-template``,
-2. ``module_name``: This is the name of your python package i.e. 'astropy' or 'sunpy'.
+1. ``package_name``: This is a human readable name for your package, like ``Astropy`` or ``SunPy``.
+2. ``module_name``: This is the name of your python package i.e. ``astropy`` or ``sunpy``.
 3. ``short_description``: This is a one sentence description of your package.
-4. ``long_description``: This is a multi-line description of your package.
-5. ``author_name``: The name or names of the authors.
-6. ``author_email``: A contact email for the authors.
-7. ``license``: The license of your project.
-8. ``project_url``: Project website.
-9. ``include_example_code``: This includes a set of example python and cython files showing you how to use the package template. If you choose 'n' then none of this will be included and you will have to populate the directory structure before you can import the package.
+4. ``author_name``: The name or names of the authors.
+5. ``author_email``: A contact email for the authors.
+6. ``license``: The license of your package.
+7. ``project_url``: Project website.
+8. ``include_example_code``: This includes a set of example python files showing you how to use the package template. If you choose ``n`` then none of this will be included and you will have to populate the directory structure before you can import the package.
+9. ``use_compiled_extensions``: Whether you plan to use compiled extensions in your package
 10. ``include_cextern_folder``: The cextern folder should be used if you are including non-python C code.
-11. ``edit_on_github_extension``: Set to "True" to enable the edit on GitHub sphinx extension.
+11. ``edit_on_github_extension``: Set to ``True`` to enable the edit on GitHub sphinx extension.
 12. ``github_project``: This is the GitHub identifier for the edit on GitHub extension and the changelog link extension.
-13. ``project_version``: The current version number of your project.
-14. ``use_travis_ci``: If 'y' the template will include an example ``.travis.yml`` file for the Travis CI service.
-15. ``use_read_the_docs``: If 'y' the ``read_the_docs.yml`` and ``.rtd-environment.yml`` files will be included for using conda on Read the Docs.
-16. ``sphinx_theme``: The value of the ``html_theme`` variable in the sphinx configuration file.
-17. ``minimum_python_version``: Version string of minimum supported Python version
+13. ``use_travis_ci``: If ``'y'`` the template will include an example ``.travis.yml`` file for the Travis CI service.
+14. ``use_read_the_docs``: If ``'y'`` the ``read_the_docs.yml`` and ``.rtd-environment.yml`` files will be included for using conda on Read the Docs.
+15. ``sphinx_theme``: The value of the ``html_theme`` variable in the sphinx configuration file.
+16. ``initialize_git_repo``: If `gitpython <https://gitpython.readthedocs.io/en/stable/>`_ is installed this option will turn the rendered package into a git repository and add and initilize the ``astropy_helpers`` submodule.
```

Comment:
I forgot to remove that :) I renumbered the questions hence why it looks like I added these lines. In fact I'm going to go with @Cadair's suggestion of removing the git option altogether since it's not that useful without astropy-helpers.